API.md

```diff
@@ -2,49 +2,21 @@
 
 The current API is described in details [here](https://lycheeorg.github.io/docs/api.html)
 
-# Module
+## v2
```

Comment:
```suggestion
## v3
```
:')

API v2 was moving:
 - from Lychee version 3 (`php/index.php?module=...&...`)
 - to Lychee version 4 (Laravel) `/api/Album::get`